src/status_im/components/toolbar_new/view.cljs

```diff
@@ -1,33 +1,114 @@
 (ns status-im.components.toolbar-new.view
-  (:require [re-frame.core :refer [subscribe dispatch]]
+  (:require [reagent.core :as r]
+            [re-frame.core :as rf]
             [status-im.components.react :as rn]
-            [status-im.components.sync-state.gradient :refer [sync-state-gradient-view]]
+            [status-im.components.sync-state.gradient :as sync-state-gradient-view]
```

Comment:
A bit odd that it is `sync-state-gradient-view/sync-state-gradient-view`- can we clarify what the namespace and function is responsible? E.g. something like `sync-state/gradient-view` would make more sense to me (without looking at the code in detail).

Reply:
True. This is a general problem with `components`.